### ISSUE TITLE
problems on load the pretrained_weights

### ISSUE THREAD

**Issue by zouliangyu** [YunYang1994/tensorflow-yolov3#35]
utils.load_weights(...., ) make the loss "NAN", do you have this issue?

**Comment by YunYang1994** [comment]
darknet53 weight?

**Comment by zouliangyu** [comment]
> darknet53 weight?

yes, darknet53.conv.74

**Comment by zouliangyu** [comment]
> > darknet53 weight?
> 
> yes, darknet53.conv.74

when I trained the COCO,  conf loss and class loss are NAN

**Comment by YunYang1994** [comment]
why not try decreasing your learning rate value ?

**Comment by zouliangyu** [comment]
> why not try decreasing your learning rate value ?

I have do that，but it does not work,

**Comment by YunYang1994** [comment]
(1) Apply the two-stage training strategy:

First stage: Restore darknet53_body part weights from COCO checkpoints, train the yolov3_head with big learning rate like 1e-3 until the loss reaches to a low level, like less than 1.

Second stage: Restore the weights from the first stage, then train the whole model with small learning rate like 1e-4 or smaller. At this stage remember to restore the optimizer parameters if you use optimizers like adam.

from https://github.com/wizyoung/YOLOv3_TensorFlow

**Comment by zouliangyu** [comment]
> (1) Apply the two-stage training strategy:
> 
> First stage: Restore darknet53_body part weights from COCO checkpoints, train the yolov3_head with big learning rate like 1e-3 until the loss reaches to a low level, like less than 1.
> 
> Second stage: Restore the weights from the first stage, then train the whole model with small learning rate like 1e-4 or smaller. At this stage remember to restore the optimizer parameters if you use optimizers like adam.
> 
> from https://github.com/wizyoung/YOLOv3_TensorFlow

Thank you, I will try that.
